game/configuration.cc

```diff
@@ -155,8 +182,8 @@ void ConfigItem::selectEnum(std::string const& name) {
 }
 
 
-std::string ConfigItem::getEnumName() {
-	int val = i();
+std::string const ConfigItem::getEnumName() const {
+	int const& val = i();
 	if (val >= 0 && val < int(m_enums.size())) return m_enums[val];
```

Comment:
Can be shortened with `val >= 0 && val < int(m_enums.size()) ? m_enums[val]; : std::string();` or does c++ not use this notation often? If not please add some braces.